CODEOWNERS

```diff
@@ -105,7 +105,7 @@
 /bundles/org.openhab.binding.fronius/ @trokohl
 /bundles/org.openhab.binding.fsinternetradio/ @paphko
 /bundles/org.openhab.binding.ftpupload/ @paulianttila
-/bundles/org.openhab.binding.gardena/ @gerrieg
+/bundles/org.openhab.binding.gardena/ @gerrieg @andrewfg
```

Comment:
Please move in separates PRs your assignation as maintainer for other bindings than Velux.

Reply:
Ok. Done.